src/config/config.js

```diff
@@ -190,5 +190,24 @@ export default {
    * @memberof module:config
    * @property {Array} known_notations
    */
-  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"]
+  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"],
+  /**
+   * use external indutny/elliptic library
+   */
+  external_indutny_elliptic: false,
+  /*
+   * use indutny/elliptic library
+   */
+  use_indutny_elliptic: true,
+  /**
+   * Path to load elliptic from external file
+   */
+  external_indutny_elliptic_path: 'elliptic.min.js',
```

Comment:
Pretty nitpicky but I would drop the "external", since it's pretty obvious that a path refers to an external file.
```suggestion
  indutny_elliptic_path: './elliptic.min.js',
```

Reply:
hmm, not obvious for me. There could be internal path, leading to node_modules for example.

Reply:
That would still be external (to the dist file) right? I.e. `external_indutny_elliptic` would be true. Also, I'm not entirely sure that loading indutny externally from node_modules using a path on Node.js is very important, as file size is not a concern there and `require('elliptic')` should just work in any case.